test/pyrostest/add_one.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import rospy
+from std_msgs.msg import Int32
+
+pub = rospy.Publisher('add_one', Int32, queue_size=10)
```

Comment:
what's the logic behind having test nodes under `test/pyrostest`? specifically why the `pyrostest`?